### PR TITLE
Documents dual authz with Mattermost

### PR DESCRIPTION
**Pull request by klizhentas**

### CODE REVIEW

File: docs/pages/access-controls/guides/dual-authz.mdx [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+title: Dual Authorization
+description: Dual Authorization for SSH and Kubernetes
+---
+
+# Dual Authorization
+
+You can set up Teleport to require require approvals of multiple members to perform some critical actions.
+Here are the most common scenarios:
+
+- Improve security of your system and prevent one successful phishing attack to compromise your system.
```
Comment (r0mant) [inline docs/pages/access-controls/guides/dual-authz.mdx:11]: "from compromising your system" sounds a bit better.

File: docs/pages/access-controls/guides.mdx [hunk 2]
```diff
@@ -9,3 +9,4 @@ description: Detailed guides for configuring Teleport Access Controls
 - [Create certs for CI/CD using impersonation](./guides/impersonation.mdx)
 - [Second Factor - U2F](./guides/u2f.mdx)
 - [Per-session MFA](./guides/per-session-mfa.mdx)
+- [Dual Authorization with ChatOps](./guides/dual-authz.mdx)
```
Comment (r0mant) [inline docs/pages/access-controls/guides.mdx:12]: Did you mean Mattermost, or wanted to keep it more generic ChatOps and potentially add other chat software to the guide later?

File: docs/pages/access-controls/guides/dual-authz.mdx [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+      - resources: ['access_plugin_data']
+        verbs: ['update']
+    # teleport currently refuses to issue certs for a user with 0 logins,
+    # this restriction may be lifted in future versions.
+    logins: ['access-plugin-not-used']
+```
+
+<Admonition
+  type="note"
+  title="Audit"
```
Comment (r0mant) [inline docs/pages/access-controls/guides/dual-authz.mdx:104]: I think Title might be incorrect, maybe "Creating resources"?

File: docs/pages/access-controls/guides/dual-authz.mdx [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+If Bob has created a request using CLI, he will assume it once it has been approved.
+Bob can also assume granted access request roles using Web UI:
+
+![Teleport Assume](../../../img/access-controls/dual-authz/teleport-7-bob-assume.png)
+
+## Troubleshooting
+
+**Cert errors**
+
+You may be getting certificate errors if Teleport's auth server missing
```
Comment (r0mant) [inline docs/pages/access-controls/guides/dual-authz.mdx:271]: auth server **is** missing address

**Review by r0mant — APPROVED** [review]

File: docs/pages/access-controls/guides/dual-authz.mdx [hunk 5]
```diff
@@ -0,0 +1,287 @@
+---
+title: Dual Authorization
+description: Dual Authorization for SSH and Kubernetes
+---
+
+# Dual Authorization
+
+You can set up Teleport to require require approvals of multiple members to perform some critical actions.
```
Comment (webvictim) [inline docs/pages/access-controls/guides/dual-authz.mdx:8]:
```suggestion
You can set up Teleport to require the approval of multiple team members to perform some critical actions.
```

File: docs/pages/access-controls/guides/dual-authz.mdx [hunk 6]
```diff
@@ -0,0 +1,287 @@
...
+  Here and below follow along and create yaml resources using `tctl create -f`:
+  ```bash
+  $ tctl create -f access.yaml
+  ```
+</Admonition>
+
+
+**Export access-plugin cert**
+
+Teleport Plugin uses the access-plugin role and user to perform the approval.
```
Comment (webvictim) [inline docs/pages/access-controls/guides/dual-authz.mdx:115]:
```suggestion
Teleport Plugin uses the `access-plugin` role and user to perform the approval.
```

File: docs/pages/access-controls/guides/dual-authz.mdx [hunk 7]
```diff
@@ -0,0 +1,287 @@
...
+We export the identify files, using tctl auth sign.
+
+```bash
+$ tctl auth sign --format=tls --user=access-plugin --out=auth --ttl=720h
+```
+
+Teleport will generate `auth.crt`, `auth.key`, and `auth.cas` - a certificate, a private key, and a CA certs respectively.
```
Comment (webvictim) [inline docs/pages/access-controls/guides/dual-authz.mdx:122]:
```suggestion
Teleport will generate `auth.crt`, `auth.key`, and `auth.cas` - a certificate, a private key, and a set of CA certs respectively.
```

File: docs/pages/access-controls/guides/dual-authz.mdx [hunk 8]
```diff
@@ -0,0 +1,287 @@
...
+  $ cd teleport-mattermost
+  $ ./install
+  ```
+</TabItem>
+<TabItem label="From Source">
+  To install from source you need `git` and `go >= (=teleport.golang=)` installed.
+
+  ```bash
+  # Checkout teleport-plugins
+  $ git clone git@github.com:gravitational/teleport-plugins.git
```
Comment (webvictim) [inline docs/pages/access-controls/guides/dual-authz.mdx:140]:
```suggestion
  $ git clone https://github.com/gravitational/teleport-plugins.git
```

File: docs/pages/access-controls/guides/dual-authz.mdx [hunk 9]
```diff
@@ -0,0 +1,287 @@
...
+```bash
+$ teleport-mattermost configure > /etc/teleport-mattermost.toml
+```
+
+Update the config with Teleport address, Mattermost URL and a bot token.
+
+```toml
+# example Mattermost configuration TOML file
+[teleport]
+# For teleport cloud you can point to the proxy: "example.teleport.sh:443"
```
Comment (webvictim) [inline docs/pages/access-controls/guides/dual-authz.mdx:157]:
```suggestion
# For Teleport Cloud you can point to the proxy: "example.teleport.sh:443"
```

**Review by webvictim — APPROVED** [review]

File: docs/pages/access-controls/guides.mdx (hunk 2 above)
Comment (klizhentas) [reply docs/pages/access-controls/guides.mdx:12]: the latter, yes

**Comment by klizhentas** [comment]
will push in another branch

**Pull request closed without merging**